CHANGELOG.md

```diff
@@ -91,6 +91,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Changed the behaviour when logging evaluation step metrics to no longer append `/epoch_*` to the metric name ([#7351](https://github.com/PyTorchLightning/pytorch-lightning/pull/7351))
 
 
+- Raise `ValueError` when a `None` value is `self.log`-ed ([#7771](https://github.com/PyTorchLightning/pytorch-lightning/pull/7771))
```

Comment:
This covers not all what you're doing in this PR, right? Maybe extend it to collection support?

Reply:
nice wording `self.log`-ed :rabbit:

Reply:
Collection support is partial in this PR.

Will include it in the CHANGELOG in a following PR